lib/fileHelpers.ts

```diff
@@ -0,0 +1,12 @@
+import _ from 'lodash'
+import * as Constants from "./CONST"
+
+const isVideoFile = (fileName: string) => {
+  const lowerCaseFileName =  fileName.toLowerCase()
+  return _.some(Constants.CONST.VIDEO_EXTENSIONS, (ext => lowerCaseFileName.endsWith(ext)))
```

Comment:
```suggestion
    return Constants.CONST.VIDEO_EXTENSIONS.some((ext) => lowerCaseFileName.endsWith(ext));
```

Reply:
Lodash is deprecated in our codebase. Let's use the native Array.prototype.some()